### ISSUE TITLE
Blizzard debuffs not hidden

### ISSUE THREAD

**Issue by Aur0r4** [Dicebar/raven#8]
The blizzard-debuffs are not hidden even when "Hide Blizzard" is active in Raven config:

![image](https://user-images.githubusercontent.com/28149249/198902495-c2890625-3cff-4a1b-8a68-41ebc720ed77.png)

**Comment by Dicebar** [comment]
Have you tried this with only Raven enabled? It's likely a different addon interfering.

**Comment by Aur0r4** [comment]
Same error, if only raven is active:
![image](https://user-images.githubusercontent.com/28149249/198903431-4a5739fa-65e5-4d2b-9f3e-9c55299a2f1e.png)

I tried with a fresh Interface/WTF folder and this is what I see after login:
![image](https://user-images.githubusercontent.com/28149249/198903544-768777ef-fbfd-4a3b-bda6-74f4fc2e4d4c.png)

**Comment by Dicebar** [comment]
That's curious. And that's with the latest version downloaded from [here](https://dicebar.net/raven/)?

**Comment by Aur0r4** [comment]
Yes but I re-downloaded it just to be sure, still the same problem. I also tested it on an alt.
Deactivating "Hide Blizzard" and re-enable it does not fix it. Blizzard-Buffs are getting disabled fine, but not the debuffs.

**Comment by Dicebar** [comment]
Debuffs specifically then, gotcha! I'll have to do some more tests on that then.

Thanks for the extremely helpful information! 👍

**Comment by Aur0r4** [comment]
Tell me if I can help somehow and thanks for your work here.

**Comment by Aur0r4** [comment]
Hello @Dicebar,

thanks for your efforts here. I've just downloaded the version from 31th October but I was still seeing the Blizzard-Debuffs.
So I tried it with a clean Interface and WTF folder again and this time the error was gone!

I found out that another addOn (PitBull Unit Frames) had a "Hide Blizzard Frames" feature too. So I disabled the "hide Buffs/debuffs" from PitBull and everything is working fine now.

Thanks for your help and keep it up! o/
